src/css/components/grid.scss

```diff
@@ -24,3 +24,82 @@
     }
   }
 }
+
+/* stylelint-disable declaration-no-important  */
+%row {
```

Comment:
This is all for the the utility classes — which are all meant to be overrides. I imagine in most cases you might not use these tools for production components, but if you did, you'd want it to override. In a world with 100% utility classes, the important would be redundant, tho.